core/src/main/java/cz/xtf/core/bm/BuildManager.java

```diff
@@ -28,7 +28,7 @@ public BuildManager(OpenShift openShift) {
                 // Otherwise you can see:
                 // $ oc label namespace <name> "label1=foo"
                 // Error from server (Forbidden): namespaces "<name>" is forbidden: User "<user>" cannot patch resource "namespaces" in API group "" in the namespace "<name>"
-                OpenShifts.admin().namespaces().withName(openShift.getNamespace())
+                OpenShifts.admin(openShift.getNamespace()).namespaces().withName(openShift.getNamespace())
```

Comment:
Must caller of `OpenShifts.admin()` provide namespace. Couldn't `OpenShifts.admin()` find out itself correct namespace?

Reply:
Seems to me it is happening already. https://github.com/xtf-cz/xtf/pull/490/files#diff-9116143614e7b1c4e2f4cf8be51914960b369af36c9ff4e20a46cf68f95cf7cdR33

Reply:
Seems to me it is happening already. https://github.com/xtf-cz/xtf/pull/490/files#diff-9116143614e7b1c4e2f4cf8be51914960b369af36c9ff4e20a46cf68f95cf7cdR33